src/sidebars/search/search-controller.js

```diff
@@ -9,6 +9,9 @@
             this.selectionService = selectionService;
             this.currentWindowService = currentWindowService;
 
+            this.quandlService.on('CONNECTION_STATUS_CHAGED', this._setConnectionStatus, this);
```

Comment:
it'd look terrible if you had a few of those together though.
Would also be weird reusing code, imo this format makes it nice and straightforward.